src/commands/install.yml

```diff
@@ -11,6 +11,7 @@ steps:
   - run:
       name: Install terraform binary
       command: |
+        cd $(mktemp -d -t tf-install-XXXXXXXXXX)
```

Comment:
Do we need to randomise this? I think it would be fine to just use something like `/tmp/tf-install`.

Reply:
We don't need to, no. I can change it to a static path. Any reason not to?